include/dfu/dfu_target.rst

```diff
@@ -39,6 +39,11 @@ For bootloader updates, the new firmware is placed in the non-active partition (
 When the complete transfer is done, call the :cpp:func:`dfu_target_done` function to mark the firmware as ready to be booted.
 On the next reboot, the device will run the new firmware.
 
+.. note::
+   If you want to have support for storing the write progress throughout a reboot of the device :option:`CONFIG_SETTINGS` and :option:`CONFIG_DFU_TARGET_MCUBOOT_SAVE_PROGRESS` needs to enabled.
+   The DFU MCUBoot target then uses the settings subsystem in Zephyr to store the current progress used by the :cpp:func:`dfu_target_write` function across power failures and device resets.
+   It has currently only be tested with FCB and NVS as storage backends.
```

Comment:
This sentence can very quickly go stale. I suggest removing it and letting sample and test configurations speak for themselves.

Reply:
Okei

Reply:
Removed